README.md

```diff
@@ -6,7 +6,7 @@
 [![Dependency Status](https://dependencyci.com/github/coredump-ch/spaceapi-rs/badge)](https://dependencyci.com/github/coredump-ch/spaceapi-rs)
 
 This is an implementation of the [SpaceAPI](http://spaceapi.net/) v0.13 in Rust
-(nightly). It contains both the type definitions as well as tools for
+(1.15+). It contains both the type definitions as well as tools for
```

Comment:
For me 1.15+ is a strange notation. Maybe something like >=1.15.0, or simply something like: "It requires Rust 1.15.0 or newer".